client/src/pages/ProfileSettings/ProfileSettings.tsx

```diff
@@ -0,0 +1,51 @@
+import { Container, Grid, Link } from '@material-ui/core';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { useState } from 'react';
+import Availability from './../../components/ProfileSettings/Availability/Availability';
+import EditProfile from './../../components/ProfileSettings/EditProfile/EditProfile';
+import Payment from './../../components/ProfileSettings/Payment/Payment';
+import ProfilePhoto from './../../components/ProfileSettings/ProfilePhoto/ProfilePhoto';
+import Security from './../../components/ProfileSettings/Security/Security';
+import Settings from './../../components/ProfileSettings/Settings/Settings';
+import useStyles from './useStyles';
+
+export default function ProfileSettings(props: any): JSX.Element {
+  const Options = ['Edit Profile', 'Profile Photo', 'Availability', 'Payment', 'Security', 'Settings'];
+
+  const classes = useStyles();
+
+  const [currentSection, setCurrentSection] = useState(props.match.params.options);
+
+  const handleClick = (section: string) => {
+    setCurrentSection(section);
+    props.history.push(`/profile/${section}`);
+  };
+
+  return (
+    <Grid container className={`${classes.root}`}>
+      <CssBaseline />
+      <Grid className={`${classes.optionItems}`}>
+        {Options.map((item) => (
+          <Link
+            onClick={() => handleClick(item.replace(/\s/g, '').toLowerCase())}
+            className={`${classes.optionItem} ${
+              currentSection === item.replace(/\s/g, '').toLowerCase() && classes.selectedOptionItem
+            }`}
+            underline="none"
+            key={item}
+          >
```

Comment:
An alternative approach to this stateful component (optional):

You could leverage Material-UI's [`component` property](https://material-ui.com/api/link/) along with [`<NavLink>` component](https://reactrouter.com/web/api/NavLink) from `react-router-dom` to apply the same logic without needing to manage state yourself.